### PR TITLE
Add guide for integrating third party assets

### PR DESCRIPTION
**Pull request by IsaacMurtagh**
image

### CODE REVIEW

File: src/content/guides/integrating-third-party-asset.mdx [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+## Defining a Payment Method
+
+We require a way of identifying your asset in order to route payments to the correct asset provider. You must define a payment method namespace, description, and list of supported currencies for us to support your asset as a payment method.
+
+### Namespace
+
+The namespace is a unique string to identify your asset. Your namespace must conform to the following properties:
+
+- Alphanumeric characters.
+- No punctuation except for ``.
```
Comment (ncjones) [inline src/content/guides/integrating-third-party-asset.mdx:21]:
```suggestion
- No punctuation except for `-`.
```

**Comment by ncjones** [comment]
@IsaacMurtagh We should lint our markdown files. What I always do when editing markdown:

* use a table mode for automatically aligning markdown tables
* use hard wrapping of paragraphs at 80 chars
* use multiline yaml strings for fields over 80 chars in frontmatter

File: src/content/guides/integrating-third-party-asset.mdx [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+
+See endpoint spec for possible failure reasons.
+
+### Pay Endpoint
+
+This endpoint is used to initiate payment. If payment is completed asynchronously, the status will be `pending` and the [get transaction endpoint](https://www.notion.so/Integrating-a-third-party-asset-v2-523b91cdea034dc08187b63341f8f1e4) will be used to poll until the status is `successful` or `failed`.
+
+```mermaid
+sequenceDiagram
+	participant P as Patron
```
Comment (ncjones) [inline src/content/guides/integrating-third-party-asset.mdx:159]: @tom-blake I knew we would regret using single letter variables in the first mermaid example

File: src/content/guides/integrating-third-party-asset.mdx [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+      "use": "sig",
+      "alg": "ES256"
+    }
+  ]
+}
+```
+
+### Error Codes
+
+You may define the errors thrown from each endpoint. If an error is received, Centrapay will [Get Transaction Status](https://www.notion.so/Get-Transaction-Status-5edc5f845cb24fc595989476a8018b1b) and decide whether to retry based on the response.
```
Comment (ncjones) [inline src/content/guides/integrating-third-party-asset.mdx:108]: notion reference

File: src/content/guides/integrating-third-party-asset.mdx [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+| successful | The transaction has been successfully processed. |
+| failed | The transaction has been unable to be successfully processed. A failure reason is expected to be provided when status is failed. |
+
+### Failure Reasons
+
+See endpoint spec for possible failure reasons.
+
+### Pay Endpoint
+
+This endpoint is used to initiate payment. If payment is completed asynchronously, the status will be `pending` and the [get transaction endpoint](https://www.notion.so/Integrating-a-third-party-asset-v2-523b91cdea034dc08187b63341f8f1e4) will be used to poll until the status is `successful` or `failed`.
```
Comment (ncjones) [inline src/content/guides/integrating-third-party-asset.mdx:155]: notion reference

File: src/content/guides/integrating-third-party-asset.mdx (hunk 2 above)
Comment (IsaacMurtagh) [reply src/content/guides/integrating-third-party-asset.mdx:159]: Is there a preference for full or shortened names?

File: src/content/guides/integrating-third-party-asset.mdx [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+	"transactionId": "UttDGTHjr7DgKoKwWpTKb",
+	"type": "payment",
+	"status": "failed",
+	"failureReason": "PAYMENT_REQUEST_EXPIRED"
+}
+```
+
+**Failure Reasons**
+
+| Name | Description |
```
Comment (alexBinns) [inline src/content/guides/integrating-third-party-asset.mdx:307]: This one too

File: src/content/guides/integrating-third-party-asset.mdx [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+	"type": "payment",
+	"status": "successful",
+	"refundable": true,
+	"refundBefore": "2023-06-09T00:52:22.468Z"
+}
+```
+
+**Failure Reasons**
+
+| Name | Description |
```
Comment (alexBinns) [inline src/content/guides/integrating-third-party-asset.mdx:217]: This table needs tidying up

**Review by daallen303 — COMMENTED** [review]
image
The small guides at the top seems odd to me

File: src/content/guides/integrating-third-party-asset.mdx [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+  path: Reference/Centrapay Integrations
+  order: 2
+---
+By integrating with Centrapay as a third-party asset provider, you can take advantage of our connections with terminals, point-of-sale systems, and merchant networks, thereby expanding the reach of your digital asset to a wider audience.
+
+Once you have defined your asset with Centrapay and implemented the required Uplink APIs, authorised merchants will be able to accept your asset as a payment option, and consumers will be able to follow the payment flow and use your asset to complete payments.
+
+## Defining a Payment Method
+
+We require a way of identifying your asset in order to route payments to the correct asset provider. You must define a payment method namespace, description, and list of supported currencies for us to support your asset as a payment method.
```
[the remark this answers is not in the document]
Comment (IsaacMurtagh) [reply src/content/guides/integrating-third-party-asset.mdx:15]: I'll make that change

File: src/content/guides/integrating-third-party-asset.mdx [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+
+**Example Request**
+
+```bash
+curl -X POST https://your.endpoint/pay \
+	-H "Authorization: ${jwt}" \
+	-H "Content-Type: application/json" \
+	-d '{
+		"currency": "NZD",
+    "amount": "1000",
```
[the remark this answers is not in the document]
Comment (IsaacMurtagh) [reply src/content/guides/integrating-third-party-asset.mdx:189]: Should have been tabs, not spaces - fixed

**Review by alexBinns — APPROVED** [review]

**Pull request merged**